src/frontend/components/SøknadsSteg/OmBarnet/useOmBarnet.tsx

```diff
@@ -253,19 +257,20 @@ export const useOmBarnet = (
         fjernPeriode: fjernBarnetrygdsperiode,
         leggTilPeriode: leggTilBarnetrygdsperiode,
         registrertePerioder: registrerteEøsBarnetrygdsperioder,
-    } = usePerioder<IEøsBarnetrygdsperiode>(
-        gjeldendeBarn.eøsBarnetrygdsperioder,
-        { mottarEllerMottokEøsBarnetrygd },
-        avhengigheter => avhengigheter.mottarEllerMottokEøsBarnetrygd.verdi === ESvar.JA,
-
-        (felt, avhengigheter) => {
+    } = usePerioder<IEøsBarnetrygdsperiode>({
+        feltId: `${BarnetrygdperiodeSpørsmålId.barnetrygdsperiodeEøs}-${barnetsUuid}`,
```

Comment:
Litt ute av konteksten her nå, men det stemmer at persontype ikke skulle være med her? 🤔

Reply:
Ja, siden det gjelder kun for det spesifikke barnet trenger vi ikke persontypen, vil ikke dukke opp spørsmål først om søker til barnet får barnetrygd, og deretter andre forelder får det.